### ISSUE TITLE
AC Mirage Flickering shadows

### ISSUE THREAD

**Issue by Rhatz** [HansKristian-Work/vkd3d-proton#1821]
Please describe your issue as accurately as possible. Include screenshots or videos if relevant

### Software information
Assassins Creed Mirage

### System information
- GPU: Rx 6700 xt
- Driver: Mesa 23.2.1
- Wine version: latest
- VKD3D-Proton version: latest

im on
lutris 0.5.14
mesa 23.2.1
kernel 6.6.6.1
ryzen 5 5600x
rx 6700 xt
4x8 (32gb)
the game is on nvme
the game is on 1.0.6
trouble shooting i already did
play on both wayland and x11
change screen to borderless, window and full screen
limit fps to 45-55-60
change refresh rate to 60 and 165(native)
i already tried playing it on steam using proton experimental and lates 8.25
i tried on windows and it works
but my main machine is on linux opensuse tumbleweed latest

here's a video sample:
https://www.youtube.com/watch?v=qWN38P_f12w

**Comment by doitsujin** [comment]
Have you tried a more up to date mesa version? We're on 23.3.2 at this point.

**Comment by Rhatz** [comment]
> Have you tried a more up to date mesa version? We're on 23.3.2 at this point.

yeah i even tried using different distro like nobara 39 with the latest kernel and mesa
sadly same

**Comment by Blisto91** [comment]
@Rhatz Is this issue still present?
